internal/init/init.go

```diff
@@ -85,40 +86,31 @@ func (c *Client) NewCommand() *cobra.Command {
 			fmt.Println()
 			token := string(b)
 			c.Servicer.SetToken(token)
-			metalClient := c.Servicer.API(cmd)
-			c.UserService = metalClient.Users
-			c.ProjectService = metalClient.Projects
+			metalGoClient := c.Servicer.MetalAPI(cmd)
+			c.UserService = *metalGoClient.UsersApi
+			c.ProjectService = *metalGoClient.ProjectsApi
 
-			user, _, err := c.UserService.Current()
+			user, _, err := c.UserService.FindCurrentUser(context.Background()).Execute()
 			if err != nil {
 				return err
 			}
-			organization := user.DefaultOrganizationID
-			project := ""
-			if user.DefaultProjectID != nil {
-				project = *user.DefaultProjectID
-			}
+			organization := user.AdditionalProperties["default_organization_id"]
+
 			fmt.Printf("Organization ID [%s]: ", organization)
 
 			userOrg := ""
 			fmt.Scanln(&userOrg)
 			if userOrg == "" {
-				userOrg = organization
+				userOrg = fmt.Sprintf("value: %v", organization)
 			}
 
-			// Choose the first project in the preferred org
-			if project == "" {
-				project, err = getFirstProjectID(c.ProjectService, userOrg)
-				if err != nil {
-					return err
-				}
-			}
-			fmt.Printf("Project ID [%s]: ", project)
+			project := user.AdditionalProperties["default_project_id"]
+			fmt.Printf("Default Project ID [%s]: ", project)
 
 			userProj := ""
 			fmt.Scanln(&userProj)
 			if userProj == "" {
-				userProj = project
+				userProj = fmt.Sprintf("value: %v", organization)
```

Comment:
The `value: ` prefix should be deleted and this should be set to a project ID so that the config file is valid.
```suggestion
				userProj = project
```

Reply:
It is shows following error "cannot use organization (variable of type interface{}) as string value in assignment: need type assertion"

Reply:
Here is the output
```
vasu@vasu-cloud:~/Equinix/metal-cli/bin$ cat /home/vasu/.config/equinix/metal.yaml
---
organization-id: *****************
project-id: ****************
token: *************
```

Reply:
@vasubabu you will need to delete that API token in the Metal Console UI ASAP so that no one can use it and then come back and remove it from your comment as well.

Reply:
it's not a original token. I will remove it